Dockerfile

```diff
@@ -6,9 +6,9 @@ ENV PIP_DEFAULT_TIMEOUT 60
 
 # Some of these packages are to have native installs so that arm packages will not be built
 RUN                                                         \
-    set -ex;                                                \
-    useradd -m buildrunner;                                 \
-    apt update;                                             \
+    set -ex &&                                              \
```

Comment:
Remove this line then, since it does what the `&&` is effectively doing.

Reply:
Or at least the `-e` portion, so it's `set -x`

Reply:
fixed